pkg/cmd/attestation/verification/policy.go

```diff
@@ -49,3 +49,57 @@ func (c EnforcementCriteria) Valid() error {
 	}
 	return nil
 }
+
+func (c EnforcementCriteria) BuildPolicyInformation() string {
+	template :=
+		`
+The following policy criteria will be enforced against all attestations:
+- Predicate type must match %s
+- Signing certificate's OIDC issuer must match %s
+- Artifact must be built in an organization whose URI is %s`
+
+	info := fmt.Sprintf(template, c.PredicateType, c.Certificate.Issuer, c.Certificate.SourceRepositoryOwnerURI)
+
+	if c.Certificate.SourceRepositoryURI != "" {
+		info += fmt.Sprintf("\n- Artifact built in a repository whose URI is %s", c.Certificate.SourceRepositoryURI)
+	}
+
+	if c.Certificate.RunnerEnvironment == GitHubRunner {
+		info += "\n- Attestation must have been generated by an Action workflow executed in a GitHub hosted runner"
+	}
+
+	if c.SAN != "" {
+		info += fmt.Sprintf("\n- Signing certificate must have a Subject Alternative Name matching the exact value %s", c.SAN)
+	} else if c.SANRegex != "" {
+		info += fmt.Sprintf("\n- Signing certificate must have a Subject Alternative Name matching the regex %s", c.SANRegex)
```

Comment:
In addition to making this match the preceding comment, a thought that occurred to me is that it might read better if all of the 

"Signing certificate foo…"
"Signing certificate bar…"
"Signing certificate baz…"

came sequentially, rather than being mixed in with other statements